app/src/main/res/values-fr/sura_names.xml

```diff
@@ -1,5 +1,122 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+  <string-array name="sura_names">
+    <item>AL-FÃTIHAH</item>
```

Comment:
@nacer80 are these intentionally caps for people in France?

Reply:
We can use also lowercase, but the Mushaf printed by KFGQPC they used uppercase.
![Screenshot_20211104-212749_Adobe Acrobat.jpg](https://user-images.githubusercontent.com/60948528/140415124-cdabf8e1-400d-4acb-9307-b6134b3823c8.jpg)

Also here they use uppercase 
https://quranenc.com/ar/browse/french_montada

Reply:
Related: Surah names used by NOOR International: https://www.noorinternational.net/fr/translation/swarlist
i did not check if they match the proposed names here

Reply:
Noor International's is a bit different than the ones from @nacer80 - but I'll go ahead and merge. please let me know if there are any I should change afterwards.